### PR TITLE
Added missed review suggestion [1]

### PR DESCRIPTION
**Pull request by thomas-fossati**
Added missed review suggestion [1], and rearranged the surrounding prose accordingly.

[1] https://github.com/thomas-fossati/draft-ftbs-rats-msg-wrap/pull/20#discussion_r1169088805

### CODE REVIEW

File: draft-ftbs-rats-msg-wrap.md [hunk 1]
```diff
@@ -160,13 +160,15 @@ number ({{Section 12.3 of -coap}}).
...
-not profiled (e.g., `application/eat+cwt`) or if the same profile
-identifier is shared by different conceptual messages, and there is no
-further context available to the CMW consumer to decide.
+: An optional bitmap that indicates which conceptual message types are
+carried in the `value` field.  Any combination (i.e., any value between
+1 and 15, included) is allowed.  This is useful only if the `type` is
+potentially ambiguous and there is no further context available to the
+CMW consumer to decide.  For example, this might be the case if the base
+media type is not profiled (e.g., `application/eat+cwt`), if the `value`
+field contains multiple different conceptual messages (e.g.,
```
Comment (nedmsmith) [inline draft-ftbs-rats-msg-wrap.md:169]:
```suggestion
field contains multiple conceptual messages with different CM types (e.g.,
```

File: draft-ftbs-rats-msg-wrap.md [hunk 2]
```diff
@@ -160,13 +160,15 @@ number ({{Section 12.3 of -coap}}).
...
-not profiled (e.g., `application/eat+cwt`) or if the same profile
-identifier is shared by different conceptual messages, and there is no
-further context available to the CMW consumer to decide.
+: An optional bitmap that indicates which conceptual message types are
+carried in the `value` field.  Any combination (i.e., any value between
+1 and 15, included) is allowed.  This is useful only if the `type` is
+potentially ambiguous and there is no further context available to the
+CMW consumer to decide.  For example, this might be the case if the base
+media type is not profiled (e.g., `application/eat+cwt`), if the `value`
+field contains multiple conceptual messages with different types (e.g.,
```
Comment (henkbirkholz) [inline draft-ftbs-rats-msg-wrap.md:169]: would "different types" not require at two examples?

File: draft-ftbs-rats-msg-wrap.md [hunk 3]
```diff
@@ -160,13 +160,15 @@ number ({{Section 12.3 of -coap}}).
...
+`application/signed-corim+cbor`), or if the same profile identifier is
+shared by different conceptual messages.
```
Comment (henkbirkholz) [inline draft-ftbs-rats-msg-wrap.md:171]: Also, maybe example?

**Review by henkbirkholz — CHANGES_REQUESTED** [review]

File: draft-ftbs-rats-msg-wrap.md (hunk 3 above)
Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md:171]: There is no such thing that I could reference now.  Although it exists in principle, and IIRC Ned had a use case for that.

File: draft-ftbs-rats-msg-wrap.md [hunk 4]
```diff
@@ -160,13 +160,15 @@ number ({{Section 12.3 of -coap}}).
...
-identifier is shared by different conceptual messages, and there is no
-further context available to the CMW consumer to decide.
+: An optional bitmap that indicates which conceptual message types are
+carried in the `value` field.  Any combination (i.e., any value between
+1 and 15, included) is allowed.  This is useful only if the `type` is
+potentially ambiguous and there is no further context available to the
+CMW consumer to decide.  For example, this might be the case if the base
+media type is not profiled (e.g., `application/eat+cwt`), if the `value`
+field contains multiple conceptual messages with different types (e.g.,
+`application/signed-corim+cbor`), or if the same profile identifier is
```
Comment (thomas-fossati) [inline draft-ftbs-rats-msg-wrap.md:170]:
```suggestion
both reference values and endorsements within the same `application/signed-corim+cbor`), or if the same profile identifier is
```

File: draft-ftbs-rats-msg-wrap.md (hunk 3 above)
Comment (henkbirkholz) [reply draft-ftbs-rats-msg-wrap.md:171]: good enough for CfA, I think.

**Review by henkbirkholz — APPROVED** [review]

**Comment by henkbirkholz** [comment]
:ship: it

File: draft-ftbs-rats-msg-wrap.md (hunk 2 above)
Comment (thomas-fossati) [reply draft-ftbs-rats-msg-wrap.md:169]: done in [e28702a](https://github.com/thomas-fossati/draft-ftbs-rats-msg-wrap/pull/29/commits/e28702a2f2d6ad178fed1ac299f295aa4c5b6fc8)

**Pull request merged**